modules/sdk-api/test/unit/v1/wallet.ts

```diff
@@ -365,26 +365,41 @@ describe('Wallet Prototype Methods', function () {
         address,
         redeemScript,
         scriptPubKey,
-        userKeyWIF: signingKey,
+        userKeyWIF: userSigningKey,
+        bitgoKeyWIF: bitgoSigningKey,
         unsignedTxHex,
         halfSignedTxHex,
+        fullSignedTxHex,
       } = await getFixture(`${__dirname}/fixtures/sign-transaction.json`);
       const testBitgo = new BitGoAPI({ env: 'test' });
       const fakeTestWallet = new Wallet(testBitgo, {
         id: address,
         private: { safe: { redeemScript } },
       });
+      const unspents = [
+        // https://blockstream.info/testnet/api/address/2N3L9cu9WN2Df7Xvb1Y8owokuDVj5Hdyv4i/utxo
+        { value: 100000, redeemScript, script: scriptPubKey },
+        { value: 100000, redeemScript, script: scriptPubKey },
+      ];
       const halfSignedTx = await fakeTestWallet.signTransaction({
         transactionHex: unsignedTxHex,
-        signingKey,
-        unspents: [
-          // https://blockstream.info/testnet/api/address/2N3L9cu9WN2Df7Xvb1Y8owokuDVj5Hdyv4i/utxo
-          { value: 100000, redeemScript, script: scriptPubKey },
-          { value: 100000, redeemScript, script: scriptPubKey },
-        ],
+        signingKey: userSigningKey,
+        unspents,
         validate: true,
       });
       halfSignedTx.tx.should.equal(halfSignedTxHex);
+
+      const fullSignedTx = await fakeTestWallet.signTransaction({
+        transactionHex: halfSignedTxHex,
+        signingKey: bitgoSigningKey,
+        unspents,
+        validate: true,
+        fullLocalSigning: true,
```

Comment:
what does this mean? Doesn't this have the effect of calling `.build()`?

Reply:
No, it's just for verifying the count of signatures. If set, then 2 sigs are needed otherwise 1.